rpm/android-tools-hadk.spec

```
@@ -54,7 +54,7 @@ make -f %{SOURCE3} -C core/mkbootimg
 %install
 rm -rf %{buildroot}
 install -D -m 755  core/fastboot/fastboot %{buildroot}%{_bindir}/fastboot
-install -D -m 755  mer-android-chroot %{buildroot}%{_bindir}/ubu-chroot
+sed -e 's/@VERSION/%{version}/' mer-android-chroot > %{buildroot}%{_bindir}/ubu-chroot
```

```suggestion
sed -e 's/@VERSION@/%{version}/' mer-android-chroot > %{buildroot}%{_bindir}/ubu-chroot
```

untested: is absence of `install` and params safe?

---

Its fine